x-pack/plugins/maps/public/routing/routes/maps_app/maps_app_view.tsx

```diff
@@ -165,15 +246,17 @@ export class MapsAppView extends React.Component {
     });
 
     // sync globalState
-    const updatedGlobalState = { filters: filterManager.getGlobalFilters() };
+    const updatedGlobalState: { filters: Filter[]; time?: TimeRange } = {
```

Comment:
Type updatedGlobalState as MapGlobalState type from global_sync.ts

Reply:
Fixed this.